scripts/header-check.sh

```diff
@@ -25,14 +25,13 @@ if [ -n "$NEW_FILES" ]
 then
     for f in $NEW_FILES; do
         echo "Checking new file: $f"
-	INVALID_FILES+=($(grep -L --exclude={ \
-            core/src/main/resources/*,\
-            core/src/test/resources/*,\
-	    user_tools/src/spark_rapids_pytools/resources/*,\
-	    user_tools/docs/resources/*,\
-	    user_tools/tests/spark_rapids_tools_ut/resources/*,\
-	    *.csv \
-	} "Copyright (c) $YEAR" $f))
+	INVALID_FILES+=($(grep -L \
```

Comment:
Can we avoid repeating `--exclude={}` and use some kind an array to store excluded paths?

Reply:
Thanks @parthosa.  grep command would look like below during the run. So I had to append --exclude for each path. PTAL. 

```
grep -L '--exclude=core/src/main/resources/*' '--exclude=core/src/test/resources/*' '--exclude=user_tools/src/spark_rapids_pytools/resources/*' '--exclude=user_tools/docs/resources/*' '--exclude=user_tools/tests/spark_rapids_tools_ut/resources/*' '--exclude=*.csv'
```